CalibCalorimetry/HcalTPGAlgos/src/HcaluLUTTPGCoder.cc

```diff
@@ -381,6 +383,35 @@ void HcaluLUTTPGCoder::update(const HcalDbService& conditions) {
   bool newHBtp = false;
   bool newHEtp = false;
   std::vector<HcalElectronicsId> vIds = emap_->allElectronicsIdTrigger();
+
+  // Access TPParameters to get HF fine-grain thresholds
+  const HcalTPParameters* tpparameters = conditions.getHcalTPParameters();
+
+  // Read thresholds from auxi1 field
+  const uint32_t aux1 =
+      tpparameters
+          ->getAuxi1();  // aux1: 32-bit auxiliary word. Currently, only the low 16 bits are used (HF MinBias FG thresholds). The high 16 bits are reserved for future use.
```

Comment:
Can I suggest the following, for an easier reading?
```suggestion
  // Read thresholds from auxi1 field
  // aux1: 32-bit auxiliary word. Currently, only the low 16 bits are used (HF MinBias FG thresholds). The high 16 bits are reserved for future use.
  const uint32_t aux1 = tpparameters->getAuxi1();  
```

Then, if you could squash all commits into a single one it would be great